clang/lib/StaticAnalyzer/Checkers/WebKit/ASTUtils.cpp

```diff
@@ -28,6 +29,15 @@ bool tryToFindPtrOrigin(
     std::function<bool(const clang::QualType)> isSafePtrType,
     std::function<bool(const clang::Expr *, bool)> callback) {
   while (E) {
+    if (auto *DRE = dyn_cast<DeclRefExpr>(E)) {
+      auto *ValDecl = DRE->getDecl();
+      auto QT = ValDecl->getType();
+      auto ValName = ValDecl->getName();
+      if (ValDecl && (ValName.starts_with('k') || ValName.starts_with("_k")) &&
```

Comment:
This ignores kCF~ and _kCF constants.

Reply:
Do the kCF~ and _kCF constants need to be handled or are they supposed to be ignored? Add a code comment clarifying this?

Reply:
They need to be treated as safe. Will add a comment.